### PR TITLE
Made Modals out of the forms

### PR DESCRIPTION
**Pull request by Rusty89**
I made the forms modals that close when you press escape or submit them

### CODE REVIEW

File: client-app/src/app/services/services.component.ts [hunk 1]
```diff
@@ -7,7 +10,22 @@ import { Component, OnInit } from '@angular/core';
...
-  constructor() { }
+  constructor(public dialog: MatDialog) { }
+  
+  
+  
+  
+  openDialog(): void {
+    const dialogRef = this.dialog.open(ActivityFormComponent, {
+       height: '400px',
+		width: '600px',
```
Comment (RialJohnson) [inline client-app/src/app/services/services.component.ts]: Haven't looked at spec at all to know what our options are, but if we can get away with not hard  coding the size that'd be best.

Comment (Rusty89) [reply client-app/src/app/services/services.component.ts]: Shoot i thought i deleted that

**Review by RialJohnson — APPROVED** [review]
Thanks for doing the research and implementing this! I would have liked see this pushed first, but I'll probably end up just stealing your code for my PR later tonight. Unless someone else sees it and we can merge it first.

**Comment by Rusty89** [comment]
Code was put in another branch and merged, this branch is unnecessary now

**Pull request closed without merging**